### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by TheArtificialDev**

### CODE REVIEW

**Review by Roopjot19 — COMMENTED** [review]
good job

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# PresidencyDays
```
Comment (Roopjot19) [inline README.md:1]: good job

Comment (TheArtificialDev) [reply README.md:1]: this is unprofessional behaviour maam

**Review by Roopjot19 — APPROVED** [review]
hehhehehehe

**Pull request merged**